includes/wc-update-functions.php

```diff
@@ -1479,6 +1479,21 @@ function wc_update_330_image_options() {
 	}
 }
 
+/**
+ * Copy order customer_id from post meta to post_author.
+ */
+function wc_update_330_order_customer_id() {
```

Comment:
If we adjust the 1 -> 0, we may need to adjust more than these orders. Although, 0 might be used already depending on _customer_user.